.github/ISSUE_TEMPLATE.md

```diff
@@ -1,21 +1,42 @@
-<!-- This issue tracker is only for technical issues related to Bitcoin Core.
+<!--- Remove text and sections that do not apply -->
 
-General bitcoin questions and/or support requests are best directed to the Bitcoin StackExchange at https://bitcoin.stackexchange.com.
+This issue tracker is only for technical issues related to btcpd.
```

Comment:
Could be generalized, since we will have the GUI in this repo too, but this is OK for now.